src/contrib/cluster/Akka.Cluster.Tools/Client/reference.conf

```diff
@@ -94,6 +94,19 @@ akka.cluster.client {
   # Turning this setting to off or false will cause the ClusterClient messages
   # to be serialized using the ClusterClientMessageSerializer and not the default Object serializer. 
   use-legacy-serialization = on
+  
+  use-initial-contacts-discovery = false
+  
+  discovery
+  {
+    method = <method>
+    actor-system-name = null
```

Comment:
if "actor-system-name" is null or whitespace, we fall back to ActorSystem.Name

Reply:
Do we want to log a warning when this happens? Kind of seems like the user should have to specify this value given that these are two separate applications communicating typically.

Reply:
Sounds like a good idea, I'll do that